cmd/restic/cmd_dump.go

```diff
@@ -168,6 +170,18 @@ func runDump(ctx context.Context, opts DumpOptions, gopts GlobalOptions, args []
 		return errors.Fatalf("loading tree for snapshot %q failed: %v", snapshotIDString, err)
 	}
 
+	if opts.Target != "" {
+		file, err := os.OpenFile(opts.Target, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0o666)
+		if err != nil {
+			return fmt.Errorf("cannot dump to file: %w", err)
+		}
+		defer func() {
+			_ = file.Close()
+		}()
+
+		os.Stdout = file
+	}
```

Comment:
Hey @MichaelEischer, 

Can you check this area again and let me know if this is good?